src/pages/basic-management/depart/edit/Index.js

```diff
@@ -0,0 +1,204 @@
+import React, { PureComponent, Fragment } from 'react';
+import { connect } from 'dva';
+import router from 'umi/router';
+import {
+    List,
+    Card,
+    Row,
+    Col,
+    Radio,
+    Input,
+    InputNumber,
+    Progress,
+    Button,
+    Icon,
+    Dropdown,
+    Menu,
+    Avatar,
+    Modal,
+    Form,
+    DatePicker,
+    Select,
+    message,
+    Cascader,
+    Divider,
+} from 'antd';
+import PageHeaderWrapper from '@/components/PageHeaderWrapper';
+import styles from '../../../List/TableList.less';
+
+const FormItem = Form.Item;
+const { Option } = Select;
+
+@connect(({ depart, dict, loading }) => ({
+    depart,
+    dict,
+    loading: loading.models.depart,
+}))
+@Form.create()
+class departEdit extends PureComponent {
+    state = {};
+
+    componentDidMount = () => {
+        console.log(this.props.match.params.id);
+        const { dispatch } = this.props;
+        dispatch({
+            type: 'depart/get',
+            payload: this.props.match.params.id,
+        });
+        dispatch({
+            type: 'depart/fetch',
+            payload: {},
+        });
+        dispatch({
+            type: 'dict/get',
+            payload: 'departType'
+        })
+    };
+
+    handleSubmit = e => {
+        e.preventDefault();
+        this.props.form.validateFieldsAndScroll((err, values) => {
+            if (!err) {
+                const { dispatch } = this.props;
+                if (!values.pid) {
+                } else {
+                    values.pid = values.pid[values.pid.length - 1];
```

Comment:
Codacy found an issue: [Assignment to property of function parameter 'values'.](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=6006324)